docs/Deployment/realtime-analytics.md

```diff
@@ -0,0 +1,41 @@
+## Overview
+
+AI DIAL Analytics Realtime is a service that gathers prompt logs from [DIAL Core](https://github.com/epam/ai-dial-core) and publishes them to InfluxDB.
+The logs of prompts are the data a user provides to LLM and the responses the LLM gives to the user. A user can access analytical data through pre-set Grafana dashboards.
+
+**TODO**
+Update AI DIAL Analytics Realtime [README](https://github.com/epam/ai-dial-analytics-realtime/blob/development/README.md) with a link to this document.
+
+## Pre-requisites
```

Comment:
I believe we need to change the order of the sections so that a person can follow them sequentially. Installing AI DIAL Analytics Realtime requires an already installed InfluxDB, and configuring Prompt Log Collector requires a running AI DIAL Analytics Realtime.

The correct sequence, in my opinion, is as follows:
1. DIAL Core
2. InfluxDB
3. AI DIAL Analytics Realtime
4. Prompt Log Collector
5. Grafana

Reply:
done